game.cpp

```diff
@@ -503,8 +503,15 @@ void Game::conquered(const string& title, int numKills, int points) {
 
 void Game::retired(const string& title, int numKills, int points) {
   int turns = getGlobalTime().getVisibleInt();
+  int dungeonTurns = campaign->getPlayerRole() == PlayerRole::ADVENTURER? 0 : getPlayerCollective()->getLocalTime().getVisibleInt() - 2;
```

Comment:
I assume the -2 at the end is because of the initial update in the initializeModels function? Please define it in a global const value, for example: static const int initialModelUpdate = 2;